src/wallet/wallet.cpp

```diff
@@ -1475,6 +1489,15 @@ void CWallet::transactionRemovedFromMempool(const CTransactionRef& tx, MemPoolRe
                 CWalletTx& wtx = wallet_it->second;
                 if (wtx.truc_child_in_mempool == tx->GetHash()) {
                     wtx.truc_child_in_mempool = std::nullopt;
+                    // Find all wallet transactions that spend utxos from this tx
```

Comment:
This comment can be confusing - we're not actually looking for transactions spending from *this* tx. We're looking for siblings of this tx, which spend from utxos of `wtx` (which is actually this transaction's parent). Might be helpful to rename `wallet_it` to `parent_it` and update this comment.

Reply:
Done